values/encoding/int_encode.go

```diff
@@ -42,39 +41,45 @@ type intEncoder struct {
 // NewIntEncoder creates a new int encoder.
 func NewIntEncoder() IntEncoder {
 	return &intEncoder{
-		// Make buf at least big enough to hold Uint64 values.
-		buf: make([]byte, uint64SizeBytes),
+		// Make buf at least big enough to hold varint64 values.
```

Comment:
`varint64` -> `int`.

Reply:
I'm going to change it to `varint encoded 64 bit int values`.

Reply:
Changed it to `...varint encoded 64 bit int values.`